lua/catppuccin/groups/integrations/feline.lua

```diff
@@ -305,6 +305,24 @@ function M.get()
 		},
 		left_sep = invi_sep,
 	}
+
+	-- lazy.nvim plugin updates
+	components.active[1][14] = {
+		provider = require("lazy.status").updates,
+		enabled = function()
+			local lazypath = vim.fn.isdirectory(vim.fn.stdpath "data" .. "/lazy/lazy.nvim")
```

Comment:
Not every user uses this path, IMO using `pcall(require, 'lazy')` to check might be a more generic solution.